gnuradio-runtime/lib/CMakeLists.txt

```diff
@@ -68,6 +63,13 @@ GR_INCLUDE_SUBDIRECTORY(thread)
 GR_INCLUDE_SUBDIRECTORY(math)
 GR_INCLUDE_SUBDIRECTORY(controlport)
 
+########################################################################
+# Setup CTRLPORT include path, if using
+########################################################################
+if(ENABLE_CTRLPORT_THRIFT)
+    include_directories(${THRIFT_INCLUDE_DIRS})
```

Comment:
What's the reasoning behind changing the `list(APPEND ..)` to a simple `include_directories(..)` ?

Reply:
2 answers: (1) If you search through the whole of the GR repo for "include_directories", this is the only instance (that I can find) where `include_directories()` as a function is not used; I found that very odd & moved to the "correct" usage. That's because (2) `include_directories` is a function that sets a directory property; it is not a list that is supposed to be directly accessible in the manner used here.

Reply:
Haven't had a lot of time lately, sorry. In the buildbot containers I have `thrift` installed. 
If you have docker for Linux available you can locally checkout a build on them: https://github.com/gnuradio/gnuradio-buildbot/tree/master/worker

I have a kinda working thrift installation on my machine, I'll checkout configuration there.

Reply:
Ok, running cmake on your branch works on my machine. So it's something special about the compiler/cmake versions used in the build containers...